.github/cmake/ctest_build.cmake

```diff
@@ -0,0 +1,34 @@
+cmake_minimum_required(VERSION 3.8)
+
+include("${CMAKE_CURRENT_LIST_DIR}/github_ci.cmake")
+
+# Read the files from the build directory.
+ctest_read_custom_files("${CTEST_BINARY_DIRECTORY}")
+# Uncomment the line below and all ctest_submit_multi lines
+# if CTestConfig.cmake defines `drop_sites`.
+# include("${CMAKE_CURRENT_LIST_DIR}/ctest_submit_multi.cmake")
+
+# Pick up from where the configure left off.
+ctest_start(APPEND)
+
+if (CTEST_CMAKE_GENERATOR STREQUAL "Unix Makefiles")
```

Comment:
could you add a comment here reminding the reader that this isn't necessary for ninja since ninja automatically parallelizes?

Reply:
Done.